cypress/fixtures/event-signup-test-data.json

```diff
@@ -7,5 +7,6 @@
 	"lastName": "Test_user1_last_name",
 	"email": "testuser@mailsac.com",
 	"phoneNumber": "07442565656",
-	"postcode": "TF3 2BT"
+	"postcode": "TF3 2BT",
+	"eventUserEmail": "eventsignupuser@mailsac.com"
```

Comment:
Is `email` still used? why do we need both? I get that we need a different email for the mailing list, event and TTA to avoid clashes, but this is the event sign up fixtures so not sure why we need two emails in here?

Reply:
Resolved